src/components/PeoplePage.tsx

```diff
@@ -1,8 +1,42 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+
+const getPreparedPeople = (people: Person[]): Person[] => {
+  const preparedPeople = people.map(person => {
+    return {
+      ...person,
+      father: people.find(p => p.name === person.fatherName),
+      mother: people.find(p => p.name === person.motherName),
+    };
+  });
+
+  return preparedPeople;
+};
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [isEmpty, setIsEmpty] = useState(false);
```

Comment:
can be calculated based on `people` variable
```suggestion
  const isEmpty = !!people.length;
```